nexus/src/updates.rs

```diff
@@ -0,0 +1,87 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use crate::db;
+use parse_display::Display;
+use serde::Deserialize;
+use std::convert::TryInto;
+
+// Schema for the `artifacts.json` target in the TUF update repository.
+#[derive(Clone, Debug, Deserialize)]
+pub struct ArtifactsDocument {
+    pub artifacts: Vec<UpdateArtifact>,
+}
+
+#[derive(Clone, Debug, Deserialize)]
+pub struct UpdateArtifact {
+    pub name: String,
+    pub version: i64,
+    pub kind: UpdateArtifactKind,
+    pub target: String,
```

Comment:
(... I feel like I may have asked you this before, but I can't remember the answer)

What is this field? Is it a reference to the `targets.json` file somehow? If it's the name of the artifact, how is it different from `name`?

Related: Could we add some docs?

Reply:
It's the name of the target in `targets.json`, yeah. I should do a docs / "is this name sensible" pass again.